presto-hive/src/test/java/com/facebook/presto/hive/TestShowStats.java

```diff
@@ -120,19 +121,19 @@ public void testShowStatsWithoutFromFails()
     @Test
     public void testShowStatsWithMultipleFromFails()
     {
-        assertQueryFails("SHOW STATS FOR (SELECT * FROM orders, lineitem)", ".*There must be exactly one table in query passed to SHOW STATS SELECT clause");
+        assertQueryFails("SHOW STATS FOR (SELECT * FROM orders, nation_partitioned)", ".*There must be exactly one table in query passed to SHOW STATS SELECT clause");
```

Comment:
keep `lineitem`

Reply:
This test case only cares the number of tables not the actual schema,  it is OK to use nation_partitioned to avoid creating a new lineitem table only for this single test case.

Reply:
@picrinite It works to use `orders` and `nation_partitioned`, but it is confusing to a reader. orders goes together with customers and lineitem; nation goes together with region. Let's make sure the queries make sense to a casual reader.

Reply:
Updated to lineitem